astilectron.go

```diff
@@ -325,10 +325,13 @@ func (a *Astilectron) executeCmd(cmd *exec.Cmd) (err error) {
 // watchCmd watches the cmd execution
 func (a *Astilectron) watchCmd(cmd *exec.Cmd) {
 	// Wait
-	cmd.Wait()
+	err := cmd.Wait()
+	if err != nil {
+		a.l.Errorf("'%v' exited with code: %v", cmd.Path, cmd.ProcessState.ExitCode())
+	}
 
 	// Check the context to determine whether it was a crash
-	if a.worker.Context().Err() != nil {
+	if err != nil || a.worker.Context().Err() != nil {
```

Comment:
I realize now that I've left a nice bug here 😄 

It shouldn't be `a.worker.Context().Err() != nil` but `a.worker.Context().Err() == nil` instead. Could you fix this ?

I'm not really comfortable adding `err != nil ||` to the condition since fixing this bug should be enough. But I may reconsider it in the coming months if I feel it's not enough. I'll change it to your proposal then.

Reply:
Gotcha, I see now. Will do.

Reply:
And yeah I think that's sufficient.